src/main/java/com/smalaca/messagesender/controller/UserController.java

```diff
@@ -4,31 +4,44 @@
 import com.smalaca.messagesender.service.UserCrud;
 import com.smalaca.messagesender.service.UserDto;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.ModelAttribute;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
+@RequestMapping("/user")
 @RestController
 public class UserController {
+    public static final String FORBIDDEN_LOGIN = "";
 
     @Autowired
     private UserCrud userCrud;
 
-    @RequestMapping("/user/create")
-    public String createUser(@ModelAttribute UserDto userDto) {
+    @RequestMapping("/create")
+    public ResponseEntity<String> createUser(@ModelAttribute UserDto userDto) {
```

Comment:
W tym wypadku to nie ma sensu. 200 (czyli OK) jest zwracane w obu Waszych przypadkach. ReponseEntity ma sens wtedy, gdy chcecie zwrócić inny http status.